templates/commands/render.go

```diff
@@ -309,6 +299,15 @@ func (r *Render) realRun(ctx context.Context, rp *runParams) (outErr error) {
 		return err
 	}
 
+	// Commit the contents of the scratch directory to the output directory. We
+	// first do a dry-run to check that the copy will fully succeed, so we don't
+	// leave a half-done mess in the user's dest directory.
```

Comment:
Can we actually guarantee that it will succeed on the second try? I'm wondering if this is overkill.

Reply:
It's not a perfect guarantee, but at least we avoid obvious issues like "destination file already exists." It's much better than just yolo'ing the files over. I changed the wording to "likely to succeed." 

Let's not let perfect be the enemy of good here.